File: JavaScript_Courses/JavaScript_Basics/Preparation/03_Conditional_Statements_Advanced_Lab/11_Fruit_Shop.js
function fruitShop(input) {
    let fruit = input[0];
    let day = input[1];
    let quantity = Number(input[2]);

    let fruitPrice = 0;

    if (day == "Monday" || day == "Tuesday" || day == "Wednesday" || day == "Thursday" || day == "Friday") {
        if (fruit == "banana") {
            fruitPrice = 2.50;
        } else if (fruit == "apple") {
            fruitPrice = 1.20;
        } else if (fruit == "orange") {
            fruitPrice = 0.85;
        } else if (fruit == "grapefruit") {
            fruitPrice = 1.45;
        } else if (fruit == "kiwi") {
            fruitPrice = 2.70;
        } else if (fruit == "pineapple") {
            fruitPrice = 5.50;
        } else if (fruit == "grapes") {
            fruitPrice = 3.85;
        } else {
            console.log("error");
        }

    } else if (day == "Saturday" || day == "Sunday") {
        if (fruit == "banana") {
            fruitPrice = 2.70;
        } else if (fruit == "apple") {
            fruitPrice = 1.25;
        } else if (fruit == "orange") {
            fruitPrice = 0.90;
        } else if (fruit == "grapefruit") {
            fruitPrice = 1.60;
        } else if (fruit == "kiwi") {
            fruitPrice = 3.00;
        } else if (fruit == "pineapple") {
            fruitPrice = 5.60;
        } else if (fruit == "grapes") {
            fruitPrice = 4.20;
        } else {
            console.log("error");
        }
    } else {
        console.log("error");
    }

    if (fruitPrice != 0) {
        console.log((quantity * fruitPrice).toFixed(2));
    }
}

// fruitShop(["apple", "Tuesday", "2"]);

File: Python_Courses/Python_Basics/Python_PB_2024_06/03_Conditional_Statements_Advanced_Lab/11_Fruit_Shop.py
fruit = input()  # banana / apple / orange / grapefruit / kiwi / pineapple / grapes
day_of_week = input()  # Monday / Tuesday / Wednesday / Thursday / Friday / Saturday / Sunday
quantity = float(input())

is_input_valid = True
price = 0
if (day_of_week == 'Monday'
        or day_of_week == 'Tuesday'
        or day_of_week == 'Wednesday'
        or day_of_week == 'Thursday'
        or day_of_week == 'Friday'):
    if fruit == 'banana':
        price = 2.50
    elif fruit == 'apple':
        price = 1.20
    elif fruit == 'orange':
        price = 0.85
    elif fruit == 'grapefruit':
        price = 1.45
    elif fruit == 'kiwi':
        price = 2.70
    elif fruit == 'pineapple':
        price = 5.50
    elif fruit == 'grapes':
        price = 3.85
    else:
        is_input_valid = False

elif day_of_week == 'Saturday' or day_of_week == 'Sunday':
    if fruit == 'banana':
        price = 2.70
    elif fruit == 'apple':
        price = 1.25
    elif fruit == 'orange':
        price = 0.90
    elif fruit == 'grapefruit':
        price = 1.60
    elif fruit == 'kiwi':
        price = 3.00
    elif fruit == 'pineapple':
        price = 5.60
    elif fruit == 'grapes':
        price = 4.20
    else:
        is_input_valid = False
else:
    is_input_valid = False

# if price != 0:
#     total_price = price * quantity
#     print(f'{total_price :.2f}')
# else:
#     print('error')

if is_input_valid:
    total_price = price * quantity
    print(f'{total_price :.2f}')
else:
    print('error')